linux/keyman-config/keyman_config/get_kmp.py

```diff
@@ -44,6 +43,15 @@ def get_download_folder():
 	"""
 	Folder where downloaded files will be saved.
 
+	Returns:
+	    str: path of user keyman cache folder
+	"""
```

Comment:
Maybe this description (compared to `keyman_cache_dir()`)?
```
str: path where downloaded files will be saved
```

Reply:
good point, done in next commit